modules/templates/article-static.xml

```diff
@@ -102,6 +102,21 @@
 {% if textDirection == "rtl" %}#content { direction: rtl; }{% endif %}
 {% if backgroundColor %}body { background-color: {{backgroundColor}}; }{% endif %}
 {{css}}
+
+.error-message {
```

Comment:
Not sure about having the CSS for error-message to be in every template, we should have it on a single file, or maybe even when we're adding it (inline styling).

Reply:
Yeah, I think this makes sense as I was doubtful having it added for every template just like what we currently have for calendar widget. Let me have this changed.